README.md

```diff
@@ -1 +1,5 @@
-# mauricio-land
\ No newline at end of file
+# mauricio-landa
+
+1.1: CSS - The Complete Guide 2022 Edition
```

Comment:
can you please add a link to the folder? so we can use the main readme file as an index

Reply:
changed it, is the change correct?